### ISSUE TITLE
[Destro Warlock] Recurrent Ritual refunding Soul Shards for Chaos Bolt

### ISSUE THREAD

**Issue by 0xjc** [simulationcraft/simc#3167]
Steps to reproduce:
Take a destro warlock profile
Replace shoulders with

    shoulder=recurrent_ritual,id=132393

Simulate and see 75% of spent time on Chaos Bolt.

**Comment by 0xjc** [comment]
Workaround:

```
shoulder=recurrent_ritual,type=cloth,quality=legendary,ilevel=910,stats=1702int_2553sta_788crit_437mastery
```

**Comment by navv1234** [comment]
This is fixed by cf3d8c7.
